# クラスタ情報のためのメッセージ型の定義

float32[] cluster_x
float32[] cluster_y